automation/all_rust_tests.sh

```diff
@@ -19,6 +19,9 @@ fi
 
 EXTRA_ARGS=( "$@" )
 
+echo "Testing with rustc version:"
+rustc --version
```

Comment:
I added this here to be able to confirm that the circleci "run tests on beta" job actually does run them on beta, but it seems worth keeping for future debugging. (And I did a trial push to confirm that yes, it does).